docs/articles_en/about-openvino/contributing.rst

```diff
@@ -0,0 +1,169 @@
+Contribute to OpenVINO
+========================
+
+.. toctree::
+   :maxdepth: 1
+   :hidden:
+
+   contributing/code-contribution-guide
+
+OpenVINO™ is always looking for opportunities to improve and your contributions
+play a big role in this process. Here are four ways you can make OpenVINO better:
+
+- `Provide feedback <#provide-feedback>`__
+- `Contribute code changes <#contribute-code-changes>`__
+- `Improve documentation <#improve-documentation>`__
+- `Promote and support OpenVINO <#promote-and-support-openvino>`__
+
```

Comment:
- Pick up a Good First Issue?

Reply:
It is strongly highlighted in "Code Contribution Guide".